config.js

```diff
@@ -319,8 +319,11 @@ var config = {
         // shard: "shard1",
         // region: "europe",
         // userRegion: "asia"
-    }
+    },
 
+    // The google oauth client id used to automatically fetch YouTube stream
+    // keys. The client should have the YouTube Data API enabled.
+    liveStreamingClientId: null
```

Comment:
i'm confused about this...what does this get used for?  i thought the user logged in dynamically in the ui.

Reply:
The user does log in dynamically through the UI. However, there needs to be a developer application the user authorizes to access certain scopes (youtube info). When the youtube api requests are made, they are made on behalf of the user using the developer application.

Reply:
`googleApiApplicationClientId`?

Reply:
I made it ID just because I've been told in the past to use all caps. I'll ask around before merge time on what is preferred.